### PR TITLE
Add ppelton San Jose file

### LINKED ISSUE #12: Create a new file for ppelton San Jose
- [x] Create a branch
- [x] Add commits
- [x] Open a PR
- [x] Collaborate
- [ ] Seek approval
- [ ] Merge the PR

### PR DESCRIPTION
**Pull request by ppelton**
closes #12

### CODE REVIEW

**Comment by isaac817** [comment]
👍  Great Job.

File: ppelton-sanjose.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# San Jose, CA
+
+### Places to eat
```
Comment (isaac817) [inline ppelton-sanjose.md]: wow, there are so many places to eat?

File: ppelton-sanjose.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+San Francisco
+Stanford
+Really anywhere around there that's outside of San Jose
+
+# San Diego, CA
+
+### Places to eat
+Cucina Urbana
+
+### Places to visit
```
Comment (isaac817) [inline ppelton-sanjose.md:21]: yeah, i love this place.

Comment (ppelton) [reply ppelton-sanjose.md:21]: I would go back to SD just to go there :)

**Review by isaac817 — APPROVED** [review]
Changes looks good.  Merging your changes.

**Pull request merged**